sleap_io/model/video.py

```diff
@@ -0,0 +1,30 @@
+"""Data model for videos.
+
+The `Video` class is a SLEAP data structure that stores information regarding
+a video and its components used in SLEAP.
+"""
+
+from __future__ import annotations
+from attr import attrs
+from attrs import define
+from typing import Any, Tuple
+
+
+@define(auto_attribs=True)
+class Video:
+    """`Video` class used by sleap to represent videos and data associated with them.
+
+    This class is used to store information regarding a video and its components.
+    It is used to store the video's `filename`, `shape`, and the video's `backend`.
+
+    Args:
+        filename: The filename of the video.
+        shape: The shape of the video.
```

Comment:
Give a better description. What does each value of `Tuple[int, int, int, int]` represent?